sd-card/html/edit_config_param.html

```diff
@@ -138,7 +138,7 @@ <h2>Configuration</h2>
 			<td style="font-size: 80%;">
 				Enable to use demo images instead of the real camera images.<br>
 				Make sure to have a demo folder on your SD-Card!
-				Check the <a href="https://jomjol.github.io/AI-on-the-edge-device-docs/Demo-Mode" target="_blank">documentation</a> for details.
+				See Details on <a href="https://github.com/jomjol/AI-on-the-edge-device/wiki/Demo-Mode" target="_blank">Demo Mode</a>.
```

Comment:
`github.io` is correct, please check the link!

Reply:
Sorry, I seem to have messed up the rebase. This should be fixed now.